app/src/main/kotlin/com/saltedge/authenticator/features/main/MainActivity.kt

```diff
@@ -107,17 +108,11 @@ class MainActivity : LockableActivity(), ViewModelContract, SnackbarAnchorContai
         viewModel.onRestartActivityEvent.observe(this, Observer { event ->
             event.getContentIfNotHandled()?.let { super.restartLockableActivity() }
         })
-        viewModel.onShowAuthorizationsListEvent.observe(this, Observer { event ->
```

Comment:
why u rm it?

Reply:
now you have two instances of AuthorizationsList in back stack. First instantiated by nav_graph, second by main activity new model.